source/Planet.cpp

```diff
@@ -231,6 +240,25 @@ void Planet::Load(const DataNode &node)
 
 
 
+// Legacy wormhole do not have an associated Wormhole object so
+// we must auto generate one if we detect such legacy wormhole.
+void Planet::FinishLoading(Set<Wormhole> &wormholes)
+{
+	// If this planet is in multiple systems, then it is a wormhole.
+	if(!wormhole && systems.size() > 1)
+	{
+		wormhole = wormholes.Get(TrueName());
+		wormhole->LoadFromPlanet(*this);
+		Files::LogError("Warning: deprecated automatic generation of wormhole \"" + name + "\" from a multi-system planet.");
+	}
+	// If the wormhole was autogenerated we need to update it to
+	// match the planet's state.
+	else if(wormhole && wormhole->IsAutogenerated())
+		wormhole->LoadFromPlanet(*this);
+}
```

Comment:
My prior comment about logging for the event case was to have logs emitted when we hit this else if branch. I'm ok with not, but knowing "when" the branch is hit could be beneficial.

Reply:
I can provide a warning if you'd like but I'm not sure it would be useful because you already get a warning for auto-generated wormholes and if you fix that error then you won't see the update warnings anyways.